app/src/main/java/org/eyeseetea/malariacare/SettingsActivity.java

```diff
@@ -156,6 +154,35 @@ private static void bindPreferenceSummaryToValue(Preference preference) {
     /**
      * Sets the application languages and populate the language in the preference
      */
+    private static void setFontSizesLanguage(Preference preference) {
+        ListPreference listPreference = (ListPreference) preference;
+
+        CharSequence[] newEntries = new CharSequence[5];
+
+
+        newEntries[0] = getLocateString(PreferencesState.getInstance().getLanguageCode(), R.string.font_extra_small);
+        newEntries[1] = getLocateString(PreferencesState.getInstance().getLanguageCode(), R.string.font_small);
+        newEntries[2] = getLocateString(PreferencesState.getInstance().getLanguageCode(), R.string.font_medium);
+        newEntries[3] = getLocateString(PreferencesState.getInstance().getLanguageCode(), R.string.font_large);
+        newEntries[4] = getLocateString(PreferencesState.getInstance().getLanguageCode(), R.string.font_extra_large);
+
+        listPreference.setEntries(newEntries);
```

Comment:
@idelcano entries is assigned twice, one in this method and two in loadFontStyleListPreference method. loadFontStyleListPreference was created because entries and entries values for fontsizes preferences should be create from font style.
In loadFontStyleListPreference method entries translated text is not working, seems to always get the strings of English resources. I think that solution should be to do working loadFontStyleListPreference with translations and not add them twice. Can you review, please?